.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
I needed that to be able install node modules, because there were conflicting peer dependencies

Reply:
Would be interesting what peer-dependencies are conflicting.

Is this project specific to you or has NgRx outdated deps?